extensions/kubernetes-client/deployment/src/main/java/io/quarkus/kubernetes/client/deployment/DevServicesKubernetesProcessor.java

```diff
@@ -166,7 +169,8 @@ private void shutdownCluster() {
     @SuppressWarnings("unchecked")
     private RunningDevService startKubernetes(DockerStatusBuildItem dockerStatusBuildItem, KubernetesDevServiceCfg config,
             LaunchModeBuildItem launchMode, boolean useSharedNetwork, Optional<Duration> timeout,
-            BuildProducer<KubernetesDevServiceInfoBuildItem> devServicesKube) {
+            BuildProducer<KubernetesDevServiceInfoBuildItem> devServicesKube,
+            BuildProducer<KubernetesDevServiceRequestBuildItem> devServiceKubeRequest) {
```

Comment:
```suggestion
            Optional<KubernetesDevServiceRequestBuildItem> devServiceKubeRequest) {
```
Same as above: KubernetesDevServiceRequestBuildItem is meant to be used by other extension to let this extension know that we want to start the dev service of the particular flavor. Think of it as an alternative to configuration for enabling this dev sevice that extensions can use.